pkgs/development/libraries/jellyfin-ffmpeg/default.nix

```diff
@@ -0,0 +1,29 @@
+{ ffmpeg, ffmpeg-full, fetchFromGitHub, lib }:
+(ffmpeg-full.override { ffmpeg = ffmpeg; }).overrideAttrs (old: rec {
```

Comment:
```suggestion
(ffmpeg-full.override { }).overrideAttrs (old: rec {
```
ffmpeg_3 got removed.

Reply:
I pinned this to `ffmpeg_4` more because of a concern that `ffmpeg` will someday be bumped to `ffmpeg_5`, while `jellyfin-ffmpeg` may still be on version 4. Not sure if this concern is valid.

Reply:
Then it should use ffmpeg_4

Reply:
`ffmpeg_4` is pinned in the `callPackage` invocation, as I understand it. Do you suggest moving it to `jellyfin-ffmpeg/default.nix` instead?

Reply:
```suggestion
(ffmpeg-full.override { ffmpeg = ffmpeg_4; }).overrideAttrs (old: rec {
```
or keep it internal 
or just use ffmpeg_4 as input without any overrides.